Changing Zcash supply curve.rst

```diff
@@ -0,0 +1,73 @@
+::
+
+  ZIP: unassigned.
+  Title: Changing Zcash supply curve.
+  Owner: unassigned
+  Author: @kek (zcash forums)
+  Advocates: @kek (zcash forums) /  @mistfpga  (zcash forums) <steve@mistfpga.net>
+  Category: Protocol
+  Created: 2019-09-04
+  License: CC BY-SA 4.0 (Creative Commons Attribution-ShareAlike 4.0) [1]
+  
+
+Originally posted and discussed `on the Zcash Community Forum <https://forum.zcashcommunity.com/t/zip-changing-zcash-supply-curve-to-3-yearly-after-2020-halving/34894>`__.
```

Comment:
@gtank and I have determined that this draft ZIP is not complete enough to merge. It would require significantly more work on Motivation, Specification, and answering objections.